users.json

```diff
@@ -0,0 +1,533 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "v1 - demo",
+        "title": "SCIM",
```

Comment:
these should describe the preferences API, with a not that the API is a subset of SCIM